CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -167,6 +167,10 @@ function Disable-DockerDriver {
         Get-ContainerNetwork | Remove-ContainerNetwork -Force
         Start-Service docker | Out-Null
     }
+
+    # Wait for Docker service to warm up in order
+    # to avoid `HNS failed with error : Unspecified error` in docker driver.
+    Start-Sleep -Seconds 6
```

Comment:
I think we should add another sleeps to test integration. We should rather handle this case correctly.

Reply:
I've investigated this a little further. So the problem is that `Docker` service creates its own VMSwitch and a container network called `nat` and while it's doing that, our attempt to create another network will result in `HNS unspecified`. I see three ways to fix that:

1. Disable Docker's `nat` network in its config on testbeds: `{
    "bridge" : "none"
}`
2. Instead of this sleep, wait for `nat` network to appear.
3. Fix this in docker driver (not sure how to do it properly yet)

Reply:
I would go for 3. 
+ write a test case in this test suite: https://github.com/codilime/contrail-windows-docker/blob/master/hns/hns_test.go#L406

Reply:
I'll revert this commit then, and do the waiting in docker driver.